nordicsemi/__main__.py

```diff
@@ -1129,5 +1130,38 @@ def zigbee(file, jlink_snr, channel):
     time.sleep(3.0) # A delay to init the OTA Server flashed on the devboard and the CLI inside of it
     of.setup_channel()
 
+@cli.group()
+def zigbee():
+    """
+    Zigbee-related commands and utilities.
+    """
+    pass
+
+@zigbee.command(short_help='Generate the Zigbee Production Config hex file.')
+@click.argument('input', required=True, type=click.Path())
+@click.argument('output', required=True, type=click.Path())
+@click.option('--offset',
+              help='Offset at which the Production Config is located',
+              type=BASED_INT_OR_NONE)
+def production_config(input, output, offset):
```

Comment:
Underscores in click command names may end up showing as hyphens on Linux, see #168. You should either specify `name="production_config"` in zigbee.command(), or rename to not use underscore in the name.